src/pykiso/auxiliary.py

```diff
@@ -57,6 +57,7 @@ def __init__(self) -> None:
         self.is_instance = False
         self.stop_event = None
         self._aux_copy = None
+        self.connector_required = True
```

Comment:
due we use this interface since the DT implementation?

Reply:
its new since we wanted to have connectors optional